.travis.yml

```diff
@@ -164,6 +164,11 @@ matrix:
       name: 'Django 1.9 Acceptance'
       env: DJANGO_VERSION=">=1.9,<1.10" TEST_SUITE=acceptance USE_SNUBA=1
 
+    # allowed to fail
```

Comment:
You're going to have to add in item with the exact env to `allow_failures` further below the travis config.

Reply:
@joshuarli Kewl, that's exactly what I was trying to figure out :)